src/scheduler/analysis.py

```diff
@@ -418,7 +407,7 @@ def _start_result_collector(self):
         self.result_collector_process = ExceptionSafeProcess(target=self._result_collector)
         self.result_collector_process.start()
 
-    def _result_collector(self):  # pylint: disable=too-complex
```

Comment:
Nit: This change should also be a separate commit.

Reply:
I removed it because it caused this:

> src/scheduler/analysis.py:410: [bad-option-value:] Bad option value 'too-complex'

I'm not even sure why. Was this option removed in a recent pylint version?

Reply:
Hmm I dont get this error.

```
$ pylint --version
pylint 2.15.3
astroid 2.12.10
Python 3.10.6 (main, Aug 10 2022, 11:40:04) [GCC 11.3.0]
```